### ISSUE TITLE
SEO friendly its not well documented and promoted CloudFlare

### ISSUE THREAD

**Issue by mckaygerhard** [arnaudsm/raito#36]
the SEO its not well explained( #26 ) , or maybe there's a bug, cos it said: _disabled by default, since it requires a default redirection to index.html_

means that due index.html is the default rendering and the javascrip only parsed the content of the file, the web server need to provide a way to rewrite the url that always request the index.html as end point of each root web page ?

but :

i put to true `browserRoute` and `sitePAth` to "/wiki" 

* when requesting `localhost/wiki/` it renders good
* when requesting `localhost/wiki/page` it does not renders if i call from url of browser
* when requesting `localhost/wiki/page` it renders if i call from a link inside a rendered page

so due last case its called from "index.html" as you pointed, works but you never defined a more explicit method to get such results, i only got a vage idea using rewrite, event spend in CloudFlare and give more importance to a enterprise.. 

also navbar fails in rare cases:

![image](https://github.com/arnaudsm/raito/assets/1571724/e8b16d79-f7d1-4c22-acfa-8bd9376a4dbd)

**Comment by arnaudsm** [comment]
I agree, giving Cloudflare as the only example was a bit lazy from me.
Which other servers examples would you like ? Nginx and Apache ?

**Comment by mckaygerhard** [comment]
> I agree, giving Cloudflare as the only example was a bit lazy from me. Which other servers examples would you like ? Nginx and Apache ?

about SEO, i am making furthest investigation for examples and provide more explicit documentation.. **for now .. i want to implement your great wiki "engine" in my documentation site and blog! @arnaudsm**

**Comment by mckaygerhard** [comment]
oh i forgotten, i an testing in most rare servers, not only apache2 or nginx.. in fact i do not tested yet those "windoser like" flavours of servers :laughing:  that is so so well used
